EIPS/eip-4762.md

```diff
@@ -89,25 +89,50 @@ Where `tree_key` and `sub_key` are computed as `tree_key, sub_key = get_storage_
 
 #### Access events for code
 
-In the conditions below, “chunk chunk_id is accessed” is understood to mean an access event of the form
+In the conditions below, _chunk `chunk_id` is accessed_ is understood to mean an access event of the form `(address, (chunk_id + 128) // 256, (chunk_id + 128) % 256)`.
 
-```
-(address, (chunk_id + 128) // 256, (chunk_id + 128) % 256)
-```
+At each step of EVM execution, the corresponding code chunk for the current accessed bytecode is considered accessed:
+
+* For legacy code, corresponds to `PC` (global program counter).
+* For EOFv1 containers, corresponds to the bytecode by referenced by `PC` within `current_section_index`.
+
+In particular, note the following cases:
+
+* The destination of a `JUMP` or positively evaluated `JUMPI` is considered to be accessed, even if the destination is not a jumpdest or is inside pushdata.
+* The destination of a `RJUMP`, `RJUMPV`, `RETF`, or positively evaluated `RJUMPI` is considered to be accessed.
+* The destination of a `JUMPI`, or `RJUMPI` is not considered to be accessed if the jump conditional is `false`.
+* The destination of a jump is not considered to be accessed if the execution gets to the jump opcode but does not have enough gas to pay for the gas cost of executing it (including chunk access cost if the jump is the first opcode in a not-yet-accessed chunk).
+* For legacy code, the destination of a jump is not considered to be accessed if it is beyond the code (`destination >= len(code)`).
```

Comment:
Compared to deleted L102, I just made the clarification this rule makes sense for legacy code.